holotrack/HoloTrackGameBase.cs

```diff
@@ -36,6 +37,9 @@ private void load()
 
             dependencies.Cache(LocalConfig);
 
+            // Temporarily read the models in the output directory. We'll have a better support for embedded resources at a later date.
+            dependencies.Cache(FaceRecognition.Create($"{RuntimeInfo.StartupDirectory}/models"));
```

Comment:
We're getting crashes because the directory doesn't exist. You might need to create this directory. The models are trying to be loaded here as well but they don't exist. Are you copying them here?

Reply:
Yes. As a temporary measure while we haven't made the final call on how to resolve the required files. Best case scenario is if we could embed those files in the DLLs itself.

Reply:
Based on my testing looks like the models are barely being copied atm. Not sure if that's intended but I'll check later.